docs/_includes/input-chips.html

```diff
@@ -0,0 +1,344 @@
+<div id="input-chips">
+    {% include section-header.html name="input-chips" version=page.versions.input-chips %}
+
+    <p>Input chips are creators and managers of <span class="highlight">chip</span> components. They allow user search of existing chip options, additions, and removals of chips. The search/selection follows the pattern of the combobox pattern.</p>
+    
+    <h3>Basic Input Chips</h3>
+    <p>This is the simplest type of implementation and includes only the bare essentials for using the module.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="input-chips">
+                <div class="input-chips__container">
+                    <div class="input-chips__content">
+                        <div class="input-chips_items" role="group" aria-describedby="input-chips-1-heading">
+                            <span class="chip">
+                                <span id="chip-interactive-1-1-text" class="chip__text">Football</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-1-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                            <span class="chip">
+                                <span id="chip-interactive-1-2-text" class="chip__text">Basketball</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-2-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                            <span class="chip">
+                                <span id="chip-interactive-1-3-text" class="chip__text">Baseball</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-3-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                        </div>
+                        <button type="button" class="icon-btn icon-btn--small input-chips_list-toggle" aria-label="Toggle options list" aria-haspopup="listbox" aria-owns="listbox-input-chips-1">
```

Comment:
I'm not sure why we have a button here. This is a deviation from combobox pattern.

Reply:
@ianmcburnie , I tried numerous ways to retain the toggle button inside the combobox markup, but found it impossible. I can poke around for a way to do it. If possible, it will almost certainly come with a hefty amount of style overrides.

Reply:
I'm saying we don't need a button at all. The combobox just expands on focus doesn't it?

Reply:
I recall we fought tooth and nail with DS to retain the toggle button during all the design iterations because of its superior UX of providing an immediate visual indication of available options to choose from. If we rely solely on focus to expose the options, we're taking a step backwards because we're losing the immediate visual cue without the need for user action/discovery.

Reply:
I meant we can keep the chevron, it just doesn't need to be in a button.